source/upgrade.txt

```diff
@@ -0,0 +1,173 @@
+.. _node-upgrade-driver:
+
+=======================
+Upgrade Driver Versions
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn about any changes you might need to make to
+your application to upgrade your driver to a new version without loss of
+functionality.
+
+Before you upgrade, perform the following actions:
+
+- Ensure the new driver version is compatible with the {+mdb-server+} version
+  your application connects to and the version of Node.js that your
+  application runs on. See the :ref:`<node-compatibility>`
+  page for this information.
+- Address any breaking changes between the current version of the driver
+  your application uses and your planned upgrade version in the
+  :ref:`<node-breaking-changes>` section of this guide. To learn
+  more about the {+mdb-server+} release compatibility changes, see the 
+  :ref:`<node-server-support-changes>` section.
+
+.. tip::
+
+   You can minimize the amount of changes that you need to make to your
+   application when upgrading driver versions by using the
+   :ref:`{+stable-api+} <nodejs-stable-api>`.
+
+.. _node-breaking-changes:
+
+Breaking Changes
+----------------
+
+A breaking change is a modification in a convention or behavior in
+a specific version of the driver that may prevent your application from
+working as expected.
+
+The breaking changes in this section are categorized by the major
+version releases that introduced them. When upgrading driver versions,
+address all the breaking changes between the current and upgrade
+versions. For example, if you are upgrading the driver from v3.x to
```

Comment:
Similar suggestion as above. I think in this case saying something like "your current version" would clear up the potential confusion around the word "current."

```suggestion
version releases that introduced them. When upgrading driver versions,
address all the breaking changes between your current version and the planned upgrade
version. For example, if you are upgrading the driver from v3.x to
```